README.md

```diff
@@ -1,112 +1,20 @@
-# Akka-gRPC Lagom Quickstart (scala)
+# Lagom Akka gRPC Quickstart with Scala
 
-This project demonstrates the usage of [akka-grpc](https://github.com/akka/akka-grpc) into Lagom.
+## Quickstart Guide
 
-## Running
+This example is described in the [Lagom Akka gRPC Quickstart with Scala guide](https://developer.lightbend.com/guides/lagom-akka-grpc-quickstart-scala/)
 
-Using gRPC in Lagom requires adding a Java Agent to the runtime. In order to handle this setting you can start the `sbt` console
-using the `ssl-lagom` script provided that takes care of downloading and setting the agent:
+This Hello World example illustrates using Akka gRPC as the communication mechanism between two Lagom services.  
+Within 30 minutes, you should be able to download and run the example and use this guide to understand how the example is constructed.
 
-```
-./ssl-lagom
-```
+You don't need to know anything about Akka-gRPC or Lagom but feel free to refer to the 
```

Comment:
Hum, I would say such guides requires some knowledge about gRCP and Lagom. How could someone understand a guide that uses gRCP for service communication in Lagom without understanding the basics about both?

Reply:
I had the same first impression but then decided it was my goal to make the guide as self-contained as possible.

Hmmm, I think it's an overly aggressive scope so we're probably better of linking to Lagom and Akka gRPC docs over and over. I'll rephrase this sentence in this PR and we can continue stuffing the docs in further PRs.